File: Python/primeOrNot.py
def isPrime(n):
    flag = True
    div = 2
    while(div < n):
        if(n % 2 == 0):
            flag = False
        div += 1
    return flag

n = 3
s = isPrime(n)

if(s):
    print("Prime number")
else:
    print("Not a prime")

File: c/primeOrNot.c
#include<stdio.h>
#include<stdbool.h>

int isPrime(int n){
    int divisor = 2;
    while (divisor < n)
    {
        if(n % divisor == 0) {
            return false;
        }
        divisor++;
    }
    return true;
}

void main(){
    int n = 3;
    bool s = isPrime(n);

    if(s){
        printf("Prime number");
    }else{
        printf("Not a prime number");
    }
}

